samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -37,7 +37,7 @@
 public class TaskConfigJava extends MapConfig {
   // Task Configs
   private static final String TASK_SHUTDOWN_MS = "task.shutdown.ms";
-  public static final long DEFAULT_TASK_SHUTDOWN_MS = 5000L;
+  public static final long DEFAULT_TASK_SHUTDOWN_MS = 30000L;
```

Comment:
Going forward, we need to allow much more time for a container to shutdown. If the container shuts down before that - there is no impact. But for huge jobs 5 sec may be not enough.